frontend/src/components/housing/submission/SubmissionListNavigator.vue

```diff
@@ -35,7 +44,23 @@ const emit = defineEmits(['submission:delete']);
 const authzStore = useAuthZStore();
 
 // State
+const route = useRoute();
 const selection: Ref<Submission | undefined> = ref(undefined);
+const dataTable = ref(null);
+const pagination: Ref<Pagination> = ref({});
+
+// read from query params if tab is set to projects otherwise use default values
+if (!route.query.tab || route.query.tab === '0') {
+  pagination.value.rows = toNumber(route.query.rows as string) ?? 10;
+  pagination.value.order = toNumber(route.query.order as string) ?? -1;
+  pagination.value.field = (route.query.field as string) ?? 'submittedAt';
+  pagination.value.page = toNumber(route.query.page as string) ?? 0;
+} else {
+  pagination.value.rows = 10;
+  pagination.value.order = -1;
+  pagination.value.field = 'submittedAt';
+  pagination.value.page = 0;
+}
```

Comment:
Style: This could potentially be simplified by doing the following instead - should be logically equivalent:
```ts
const pagination: Ref<Pagination> = ref({
  rows: 10,
  order: -1,
  field: 'submittedAt',
  page: 0
});

// read from query params if tab is set to enquiry otherwise use default values
if (!route.query.tab || route.query.tab === '0') {
  pagination.value.rows = toNumber(route.query.rows as string) ?? 10;
  pagination.value.order = toNumber(route.query.order as string) ?? -1;
  pagination.value.field = (route.query.field as string) ?? 'submittedAt';
  pagination.value.page = toNumber(route.query.page as string) ?? 0;
}
```